sdk/lib/3rdparty/zlib/CMakeLists.txt

```diff
@@ -21,9 +21,22 @@ list(APPEND SOURCE
     infback.c
     uncompr.c)
 
+list(APPEND MINIZIP_SOURCE
```

Comment:
Maybe MiniZip deserves it's own CMakeLists.txt?

Reply:
Adding it inside an additional `add_subdirectory` would just be a pointless indirection.

What is there to see in the Makefile?